### ISSUE TITLE
`@JacksonFeature` can't be used for deserialization (not applicable to parameters)

### ISSUE THREAD

**Issue by mlewe** [FasterXML/jackson-jaxrs-providers#113]
The `@JacksonFeature` annotation cannot be used for deserialization, since it's declared as `@Target({ElementType.ANNOTATION_TYPE, ElementType.METHOD })`.

However, in JAX-RS, MessageBodyReader.readFrom will be called with the parameter annotations, not the method annotations.

As a workaround, it's possible for me to define a new annotation with `@JacksonAnnotationsInside` that can also be applied to parameters, e.g.
```java
@JacksonAnnotationsInside
@JacksonFeatures(deserializationDisable = DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
@Target(ElementType.PARAMETER)
@Retention(RetentionPolicy.RUNTIME)
@interface DontFailOnUnknownProperties {
}
```

**Comment by cowtowncoder** [comment]
That does sound like an unfortunate flaw. Will fix for 2.10 (needs to go in minor version due to change to an API, and esp. so for annotations).

**Comment by cowtowncoder** [comment]
Fixed in 2.10.0
